### PR TITLE
refactor: Add page access control to routes

### PR DESCRIPTION
**Pull request by iwisunny**

### CODE REVIEW

**Comment by iwisunny** [comment]
solve #593

File: src/routes/index.js [hunk 1]
```diff
@@ -20,56 +19,62 @@ const routes = {
...
-  '/:dash/app-review/:reviewId': Dash.ReviewDetail,
+  '/:dash/apps': [Dash.Apps, { acl: [r.admin, r.isv] }],
+  '/:dash/app/:appId': [Dash.AppDetail, { acl: [r.admin, r.isv] }],
 
-  '/:dash/apps/:appId/deploy/:versionId?': Dash.AppDeploy,
+  '/:dash/app-reviews': [Dash.Reviews, { acl: [r.admin, r.isv] }],
+  '/:dash/app-review/:reviewId': [Dash.ReviewDetail, { acl: [r.admin, r.isv] }],
 
-  '/:dash/account/:type?': Dash.Account,
+  '/:dash/account/:type?': [Dash.Account, { acl: [r.admin, r.isv] }],
```
Comment (whDongRui) [inline src/routes/index.js]: Account page should for all roles

File: src/routes/index.js [hunk 2]
```diff
@@ -20,56 +19,62 @@ const routes = {
...
+  '/:dash/app/:appId/sandbox-instance/:clusterId': [
+    Dash.ClusterDetail,
+    { acl: r.dev }
+  ],
 
   // only for isv
-  '/:dash/provider-detail': Dash.ApplicationDetail,
-  '/:dash/provider/submit': Dash.ProviderCreate,
+  '/:dash/provider-detail': [Dash.ApplicationDetail, { acl: r.isv }],
+  '/:dash/provider/submit': [Dash.ProviderCreate, { acl: r.isv }],
```
Comment (whDongRui) [inline src/routes/index.js]: ProviderCreate should for user or isv

File: src/routes/index.js [hunk 3]
```diff
@@ -20,56 +19,62 @@ const routes = {
...
 
-  '/:dash/clusters': Dash.Clusters,
-  '/:dash/cluster/:clusterId': Dash.ClusterDetail,
+  '/:dash/clusters': [Dash.Clusters, { acl: r.admin }],
+  '/:dash/cluster/:clusterId': [Dash.ClusterDetail, { acl: r.admin }],
 
-  '/:dash/runtimes': Dash.Runtimes,
-  '/:dash/runtime/create': Dash.CreateRuntime,
+  '/:dash/runtimes': [Dash.Runtimes, { acl: r.admin }],
+  '/:dash/runtime/create': [Dash.CreateRuntime, { acl: r.admin }],
```
Comment (whDongRui) [inline src/routes/index.js]: Runtime should for all roles

File: src/routes/index.js (hunk 1 above)
Comment (iwisunny) [reply src/routes/index.js]: Oh, right

File: src/routes/index.js (hunk 2 above)
Comment (iwisunny) [reply src/routes/index.js]: ok

File: src/routes/index.js (hunk 3 above)
Comment (iwisunny) [reply src/routes/index.js]: yes

**Pull request merged**